soc/arm/nordic_nrf/nrf91/power.c

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2017 Intel Corporation.
+ * Copytight (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <zephyr.h>
+#include <power/power.h>
+#include <hal/nrf_regulators.h>
```

Comment:
This file is identical to the one added for nrf53, except that this one doesn't conditionalize the include on `CONFIG_HAS_SYS_POWER_STATE_DEEP_SLEEP_1`.  It's also remarkably similar to nrf51 and nrf52, differing only in the include file and the spelling of `power_system_off`.  Could it perhaps be moved to a `soc/arm/nordic_nrf/common` directory to reduce the need to make changes in four files as the power management infrastructure starts to mature?

Reply:
Those files are almost identical now, but I have created separate ones to prepare for potential differences in power management's future implementation. As for now, they could be merged into one, I don't have a strong opinion on this.

Reply:
OK.